packages/alfa-rules/test/common/predicate/is-visible.spec.tsx

```diff
@@ -61,6 +61,37 @@ test(`isVisible() returns false when an element is hidden by reducing its size
   t.equal(isVisible(element), false);
 });
 
+test(`isVisible() returns false when an element is hidden by reducing its size
+      to 1x1 pixels and clipping overflow`, (t) => {
+  const element = (
+    <div style={{ width: "0px", height: "0px", overflow: "hidden" }}>
+      Hello World
+    </div>
+  );
```

Comment:
That one is indeed not needed since it is already covered by the previous tests.
```suggestion
```